gui/src/hooks/useHistory.tsx

```diff
@@ -54,16 +54,13 @@ function useHistory(dispatch: Dispatch) {
     if (state.history.length === 0) return;
 
     const stateCopy = { ...state };
-    if (open_new_session){
+    if (open_new_session) {
```

Comment:
Nitpick but think this should be camelCase
```suggestion
    if (openNewSession) {
```